website/content/docs/agent/config/config-files.mdx

```diff
@@ -2293,11 +2293,9 @@ tls {
       "cert_file": "/etc/pki/tls/certs/my.crt",
       "ca_file": "/etc/pki/tls/certs/ca-bundle.crt",
       "verify_incoming": true,
-      "verify_outgoing": true
-    },
-    "internal_rpc": {
+      "verify_outgoing": true,
```

Comment:
Can you also update the HCL example above this to match? https://github.com/hashicorp/consul/pull/17155/files#diff-5276790402104907939b2b89fae9709e6664b11211486deffe4da883844e1945R2258-R2270

We'd like the HCL and JSON examples to be equivalent. Thank you!